Assets/_BForBoss/_Character/Scripts/FirstPersonPlayer.cs

```diff
@@ -8,6 +8,10 @@ namespace BForBoss
 {
     public partial class FirstPersonPlayer : FirstPersonCharacter
     {
+#if UNITY_EDITOR || DEVELOPMENT_BUILD
```

Comment:
The FIrstPersonPlayer should not have any changes. Since now, this script is tightly coupled with the editor tool.

Is it crucial that the actions needs to be disabled?

Reply:
There's also another issue here. 
The character shouldn't know about this so called `DebugWindow`, 
If thet pause screen shows up, it shouldn't know about the "pause screen" either.

What it should know, is when to `freeze` or `StopAllActions`.  And it shouldn't be behind a #Debug
This is more of a naming issue

Reply:
Fair point. So would you rather let DebugWindow have an instance of FirstPersonPlayer?

Reply:
Unfortunately, we do have to Disable this, cause mouse clicking over anything not a gameobject (i.e. GUI layout buttons) will force the Cursor to lock

Reply:
So now this is handled strictly via `IInputSettings`